layout.tt

```diff
@@ -40,76 +40,39 @@
-            <li><a href="[%root%]nixos/about.html">About</a></li>
-            <li><a href="[%root%]download.html">Download</a></li>
+          <ul class="nav">
             <li><a href="[%root%]learn.html">Learn</a></li>
-            <li><a href="[%root%]nixos/packages.html">Packages</a></li>
-            <li><a href="[%root%]nixos/options.html">Options</a></li>
-            <li><a href="[%root%]nixos/community.html">Community</a></li>
+            <li><a href="[%root%]download.html">Download</a></li>
             <li><a href="[%root%]governance.html">Governance</a></li>
+            <li><a href="[%root%]community.html">Community</a></li>
           </ul>
-          <ul class="nav pull-right">
-            <!--
-                <li><a href="https://github.com/NixOS/nixpkgs"><i class="fa fa-github"></i></a></li>
-                <li><a href="https://twitter.com/nixos_org"><i class="fa fa-twitter"></i></a></li>
-                -->
-            [% ELSIF menu == 'nix' %]
-            <ul class="nav pull-left">
-              <li><a href="[%root%]nix/about.html">About</a></li>
-              <li><a href="[%root%]download.html">Download</a></li>
-              <li><a href="[%root%]learn.html">Learn</a></li>
-              <li><a href="[%root%]governance.html">Governance</a></li>
-            </ul>
-            <ul class="nav pull-right">
-              <li><a href="https://github.com/NixOS/nix"><i class="fa fa-github"></i></a></li>
-              [% ELSIF menu == 'nixpkgs' %]
-              <ul class="nav pull-left">
-                <li><a href="[%root%]download.html">Download</a></li>
-                <li><a href="[%root%]learn.html">Learn</a></li>
-                <li><a href="[%root%]governance.html">Governance</a></li>
-              </ul>
-              <ul class="nav pull-right">
-                <li><a href="https://github.com/NixOS/nixpkgs"><i class="fa fa-github"></i></a></li>
-                [% ELSIF menu == 'organization' %]
-                <ul class="nav pull-right">
-                  <li><a href="[%root%]learn.html">Learn</a></li>
-                  <li><a href="[%root%]governance.html">Governance</a></li>
-                  <li><a href="https://github.com/NixOS/nixpkgs"><i class="fa fa-github"></i></a></li>
-                  <li><a href="https://twitter.com/nixos_org"><i class="fa fa-twitter"></i></a></li>
-                  [% ELSE %]
-                  <ul class="nav pull-right">
-                    [% END %]
-
-                  </ul>
+          [% IF !hideSearch %]
+          <form class="navbar-search pull-right">
+            <div class="input-append">
+              <input class="span2" id="appendedDropdownButton" type="text" />
+              <div class="btn-group">
+                <button class="btn dropdown-toggle" data-toggle="dropdown">
+                  Search
+                  <span class="caret"></span>
+                </button>
+                <ul class="dropdown-menu pull-right">
```

Comment:
i would assume a search on the website there

Reply:
Me as well but for now we will stick to what is already present on current website. we can then discuss which search engine we want to default to.